iphone/Classes/MediaModule.m

```diff
@@ -2095,8 +2095,14 @@ - (void)imagePickerController:(UIImagePickerController *)picker_ didFinishPickin
     if (cropRect != nil) {
         [dictionary setObject:cropRect forKey:@"cropRect"];
     }
-	
+    
     [self sendPickerSuccess:dictionary];
+    
+#ifndef TI_USE_KROLL_THREAD
+    //TIMOB-24389 : iOS: Appcelerator Titanium App Crashes after some usage time
```

Comment:
Describe the issue rather then paste the ticket title. That will help others to get the details more quickly